scripts/TestHarness/testers/RavenUtils.py

```diff
@@ -90,7 +91,7 @@ def __lookup_preferred_version(name, optional=False):
                ("netcdf4", __lookup_preferred_version("netCDF4")),
                ("matplotlib", __lookup_preferred_version("matplotlib")),
                ("statsmodels", __lookup_preferred_version("statsmodels")),
-               #("tensorflow", __lookup_preferred_version("tensorflow")),
+               ("tensorflow", "1.12.0"),
```

Comment:
Libraries that are not in the default libraries should be added to the libraries that scripts/library_report reports on. (So library report should be updated to report if tensorflow is found and its version.)

Reply:
added